lib/Runtime/Base/DelayLoadLibrary.h

```diff
@@ -167,6 +167,7 @@ namespace Js
 
         Js::DelayLoadWinRtString *winRTStringLibrary;
         BOOL winRTStringsPresent;
+        bool hasGlobalizationDllLoaded = false;
```

Comment:
Kind of weird to have both BOOL and bool in the same file. would be nice to make them consistent.